src/zkevm_specs/evm/execution/extcodehash.py

```diff
@@ -0,0 +1,33 @@
+from ..instruction import Instruction, Transition
+from ..table import CallContextFieldTag, AccountFieldTag
+from ..opcode import Opcode
+from ...util.param import N_BYTES_MEMORY_ADDRESS
+from ...util import keccak256
+
+
+def extcodehash(instruction: Instruction):
+    opcode = instruction.opcode_lookup(True)
+
+    address = instruction.stack_pop()
+
+    tx_id = instruction.call_context_lookup(CallContextFieldTag.TxId)
+    is_warm = instruction.add_account_to_access_list(tx_id, address)
+
+    nonce = instruction.account_read(address, AccountFieldTag.Nonce)
+    balance = instruction.account_read(address, AccountFieldTag.Balance)
+    code_hash = instruction.account_read(address, AccountFieldTag.CodeHash)
+
+    is_empty = nonce == 0 and balance == 0 and code_hash == int.from_bytes(keccak256(""), "big")
```

Comment:
you should use `instruction.is_zero` to check if a value is 0, and multiply all three conditions together, like
`instruction.is_zero(nonce) * instruction.is_zero(balance) * instruction.is_zero(code_hash - hash)`